content/articles/how-to-implement-k-fold-cross-validation/index.md

```diff
@@ -0,0 +1,417 @@
+### How to Implement K fold Cross-Validation in Scikit-Learn
+
+The most typical strategy in machine learning is to divide a data set into training and validation sets. 70:30 or 80:20 could be the split ratio. It is the holdout method. The problem with this strategy is that we don't know if a high validation accuracy indicates a good model. What if the part of the data we utilized for validation turned out to be a success? Would our model still be accurate if we used a different section of the data set as a validation set? These are some questions that K fold CV answers. The GitHub repo for this project is [here](https://github.com/Inyrkz/breast_cancer/blob/main/k_fold_cv_article_guide.ipynb).
+
+### Prerequisites
+- Wisconsin Breast Cancer data set [here](https://www.kaggle.com/uciml/breast-cancer-wisconsin-data)
+- Google Colaboratory or Jupyter Notebook
+- Basic Knowledge of Decision Tree Algorithm [here](https://www.section.io/engineering-education/entropy-information-gain-machine-learning/)
+
+### Outline
+- [Introduction](#introduction)
+- [Data Preprocessing](#data-preprocessing)
+- [5-Fold Cross-Validation](#5-fold-cross-validation)
+- [Model Training](#model-training)
+- [Conclusion](#conclusion)
+- [References](#references)
+
+### Introduction
+K fold cross-validation is a superior technique to validate the performance of our model. It evaluates the model using different chunks of the data set as the validation set. 
+
+We divide our data set into K folds. K represents the number of folds into which you want to split your data. If we use 5-folds, the data set divides into five sections. In different iterations, one part becomes the validation set.
+
+![5 Fold Cross-Validation](/engineering-education/how-to-implement-k-fold-cross-validation/5-fold-cv.jpeg)
+
+*[Image Source: Great Learning Blog](https://www.mygreatlearning.com/blog/cross-validation/)*
+
+In the first iteration, we use the first part of the data for validation. We use the other parts of the data set for training as illustrated in the image above.
```

Comment:
Consider replacing the second part of the sentence with, "As illustrated in the image above, we use the other parts of the data set for training.	 "

Reply:
Done